File: eBarberShop.Models/Recenzija.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eBarberShop.Models
{
    public class Recenzija
    {
        public int RecenzijaID { get; set; }
        public string SadrzajRecenzije { get; set; }
        public int Ocjena { get; set; }
        public DateTime DatumKreiranja { get; set; }
        public int KorisnikID { get; set; }
    }
}

File: ebarbershop_mobile/lib/models/recenzija.dart
import 'package:flutter/cupertino.dart';
import 'package:json_annotation/json_annotation.dart';

import 'korisnik.dart';

part 'recenzija.g.dart';

@JsonSerializable()
class Recenzija{
  int? recenzijaID;
  String? sadrzajRecenzije;
  int? ocjena;
  DateTime? datumKreiranja;
  int? korisnikID;
  Korisnik? korisnik;

  Recenzija();

  factory Recenzija.fromJson(Map<String, dynamic> json) => _$RecenzijaFromJson(json);

    Map<String, dynamic> toJson() => _$RecenzijaToJson(this);
}